operator/src/main/resources/scripts/introspectDomain.py

```diff
@@ -788,12 +791,36 @@ def addNetworkAccessPoint(self, server, nap, is_server_template):
       name=self.name(nap)
     self.writeln("  - name: " + name)
     self.writeln("    protocol: " + self.quote(nap_protocol))
+    if nap.getListenPort() == 0:
+      trace("SEVERE", "Invalid listen port value '"
+            + str(nap.getListenPort())
+            + "' in the WebLogic Domain for "
+            + server.getName()
+            + ' Network Channel '
+            + nap.getName()
+            + '. Please provide a valid value for the listen port, this is likely because of not specifying the port '
+              'value during domain '
+              'creation')
+      sys.exit(1)
```

Comment:
Since this is a validation error, can this be detect this during the validation phase, and, in the validation, use 'addError' instead of a SEVERE and an abrupt sys.exit(1)?

Reply:
will move to validate